### PR TITLE
Add docs about faucet 

### PR DESCRIPTION
**Pull request by Pana**
Add three doc
1. How to use Faucet
2. How to download conflux-network client
3. How to run a miner

### CODE REVIEW

File: docs/run-a-node/en/how-to-get.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+id: how_to_get
+title: Get Conflux Client
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/run-a-node/en/how-to-get.md
+keywords:
+  - node
+---
+
+[Conflux-rust](https://github.com/conflux-chain/conflux-rust) is a Conflux protocol high-performance implementation developed by the foundation with the Rust language. It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public. It is also a command-line tool that manages local accounts and conducts PRC interaction with nodes. You can download the release package through any of the ways listed:
```
Comment (Thegaram) [inline docs/run-a-node/en/how-to-get.md:9]: > is a Conflux protocol high-performance implementation

is a high-performance implementation of the Conflux protocol

> It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public.

Conflux-rust is a Conflux node that connects to other nodes on the Conflux network and provides an RPC interface that you can use to query and interact with the blockchain.

> release package

released package

File: docs/run-a-node/en/how-to-get.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+id: how_to_get
+title: Get Conflux Client
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/run-a-node/en/how-to-get.md
+keywords:
+  - node
+---
+
+[Conflux-rust](https://github.com/conflux-chain/conflux-rust) is a Conflux protocol high-performance implementation developed by the foundation with the Rust language. It is an Conflux Client program that runs Conflux network nodes and provides RPC service to the public. It is also a command-line tool that manages local accounts and conducts PRC interaction with nodes. You can download the release package through any of the ways listed: 
+
+## Directly download the nodes program
```
Comment (Thegaram) [inline docs/run-a-node/en/how-to-get.md:11]: > Directly download the nodes program

Download pre-built binaries

File: docs/run-a-node/en/how-to-get.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Conflux-rust on Github [Release](https://github.com/Conflux-Chain/conflux-rust/releases) page provides a compiled node program that can be directly downloaded and run. There are two versions for the main-net and the test-net: the main-net version is Conflux `vx.x.x`, while the version for test-net is `Conflux vx.x.x-testnet` (which has a suffix of `testnet`). Each program will be provided for Linux, Windows10, and macOS.
```
Comment (Thegaram) [inline docs/run-a-node/en/how-to-get.md:13]: > Conflux-rust on Github [Release](https://github.com/Conflux-Chain/conflux-rust/releases) page provides a compiled node program that can be directly downloaded and run.

The [release](https://github.com/Conflux-Chain/conflux-rust/releases) page on the conflux-rust GitHub repository providers pre-built binaries that you can download and run directly.

> main-net, test-net

mainnet, testnet

File: docs/run-a-node/en/how-to-get.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+On the Release page, each version will have a version update description and `Assets` that you can download. After selecting the corresponding platform, you can click the link to download the zip package. For example, when unzipping the `conflux_linux_v1.1.2.zip` file, a `run` folder that contains node programs, configuration files, and startup scripts will appear. The specific documents include:
+
+```bash
+➜  run tree
+.
+├── conflux
+├── log.yaml
+├── start.bat
```
Comment (Thegaram) [inline docs/run-a-node/en/how-to-get.md:22]: Does the linux zip have this bat file?

File: docs/run-a-node/en/how-to-get.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+0 directories, 6 files
+```
+* `conflux` Node program
+* `tethys.toml` Mainnet configuration file (If you download the testnet program, the configuration file is testnet.toml)
+* `log.yaml` Log configuration file
+* `throttling.toml` Internet speed flow limit configuration file (used for development and testing) 
+* `start.sh` Linux system startup script
+* `start.bat` Windows system startup script
+
```
Comment (Thegaram) [inline docs/run-a-node/en/how-to-get.md:35]: Please add something like this:

**Make sure you only run binaries downloaded from the official Conflux-Chain/conflux-rust GitHub repository.**

**Review by Thegaram — CHANGES_REQUESTED** [review]
I cannot see the "How to run a miner" part

**Comment by Pana** [comment]
"How to run a miner" is on forum, here is only a link in sidebar.

**Review by Thegaram — APPROVED** [review]

**Pull request merged**